app/Rules/EmailNotExistRule.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Rules;
+
+use App\Models\User;
+use Illuminate\Contracts\Validation\Rule;
+
+class EmailNotExistRule implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        if (User::whereEmail($value)->doesntExist()) {
```

Comment:
suggestion 

maybe there is a method `exist()`? if so, the return is more optimized like ` return User::whereEmail($value)->exists();`

Reply:
Done 🙇